<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>H264Config</class>
 <widget class="QWidget" name="H264Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>659</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>659</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>H264 Configuration Panel</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>76</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>76</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>NAL Format:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboNAL">
         <property name="minimumSize">
          <size>
           <width>166</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>166</width>
           <height>24</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Startcodes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NAL Unit per Buffer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1 Byte Interleave Length</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2 Byte Interleave Length</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4 Byte Interleave Length</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dummy</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Intra Mode:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboIntra">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>24</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>4x4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8x8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16x16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dummy</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>81</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Intra Period: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineIntraPeriod">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>24</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="0" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="buttonOK">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>71</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>71</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Rate Cotrol:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboRate">
         <property name="minimumSize">
          <size>
           <width>161</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>161</width>
           <height>24</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Default</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Variable</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Constant</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Variable Skip Frames</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Constant Skip Frames</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>81</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Encode QP :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineQP">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>24</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Min Quant:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineMinQuant">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>24</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Max Quant:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineMaxQuant">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>24</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>38</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Profile:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboProfile">
         <property name="minimumSize">
          <size>
           <width>142</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>142</width>
           <height>24</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Baseline</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Main</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Extended</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>High</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>High 10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>High 422</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>High 444</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Constrained Baseline</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>41</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>41</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Level:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboLevel">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>24</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1b</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2.1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2.2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3.1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3.2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4.1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4.2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5.1</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>41</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Bitrate:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineBitrate">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>24</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>33</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>33</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>kbps</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>87</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3" rowspan="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>74</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkMinFrag">
     <property name="minimumSize">
      <size>
       <width>165</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>165</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Minimize Fragmentation</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOK</sender>
   <signal>clicked()</signal>
   <receiver>H264Config</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
